Packages/com.unity.inputsystem/InputSystem/Devices/InputDevice.cs

```diff
@@ -521,10 +521,24 @@ protected virtual void OnAdded()
         /// </summary>
         /// <remarks>
         /// This is called <em>after</em> the device has already been removed.
-        /// </remarks>
         /// <seealso cref="InputSystem.devices"/>
         /// <seealso cref="InputDeviceChange.Removed"/>
         /// <seealso cref="OnRemoved"/>
+        /// </remarks>
+        /// <example>
+        /// <code>
+        /// public class MyDevice : InputDevice
+        /// {
+        ///     protected override void OnRemoved()
+        ///     {
+        ///         // use this context to unassign the current device for instance
+        ///         base.OnRemoved();
+        ///         if (current == this)
+        ///             current = null;
+        ///     }
+        /// }
+        /// </code>
+        /// </example>
         protected virtual void OnRemoved()
```

Comment:
Do we need to add a example for `protected` methods ? I don't see how users can actually use them.

Reply:
Good question, I was also surprised to see this. @rita did the report/table contain missing example for protected method in its report?

Reply:
yes, since it's possible to create custon input devices this can be overwritten by users.

Reply:
Ah yes. Makes a lot of sense.
I've made sure to inherit this on my Gamepad PR. Thank you!